src/js/components/Poll/PublicRegisterModal.vue

```diff
@@ -190,13 +190,13 @@ export default {
 
 			if (!this.isValidName) {
 				return {
-					result: t('polls', 'Name {username} is invalid or reserved.', { username: this.userName }),
+					result: t('polls', 'User name {username} is invalid or reserved.', { username: this.userName }),
```

Comment:
Please see my last comment. User name or username should be avoided in this combination.

Reply:
@dartcafe So it should be: `User name is invalid or reserved.`?

Reply:
@Valdnet 

Username can be interpreted as a user id, which is the login name of a user. In this case we just ask for a random name, therefore is should only be called `name`, IMHO.

It is just "Name Peter is invalid or reserved." still sounds strange to me, why I prefer to say "The name Peter is invalid or reserved."

Reply:
That's what I think fits best, if the old variant isn't clear enough fot the translations.